src/runtime/virtcontainers/pkg/agent/protocols/client/client.go

```diff
@@ -237,6 +238,11 @@ func parse(sock string) (string, *url.URL, error) {
 		}
 		hybridVSockPort = uint32(port)
 		grpcAddr = HybridVSockScheme + ":" + hvsocket[0]
+	case RemoteSockScheme:
+		if addr.Host != "" {
+			return "", nil, grpcStatus.Errorf(codes.InvalidArgument, "Invalid remote sock scheme: %s", sock)
```

Comment:
I suggest to explain it is invalid because the host address should be nil, if I understood correctly :)

Reply:
OK. I'll add such explanation in the error message.

Reply:
done